internal/cli/login/oauth.go

```diff
@@ -14,15 +14,23 @@ import (
 	"github.com/pkg/browser"
 )
 
-type GetClientFunc func(callbackURL string) (*osincli.Client, error)
+type GetClientFunc func(callbackURL string) (*osincli.Client, string, error)
 
-func getOAuth2AccessToken(client *osincli.Client, authorizeRequest *osincli.AuthorizeRequest, r *http.Request) (AuthInfo, error) {
+func getOAuth2AccessToken(client *osincli.Client, clientId string, authorizeRequest *osincli.AuthorizeRequest, r *http.Request) (AuthInfo, error) {
 	areqdata, err := authorizeRequest.HandleRequest(r)
 	if err != nil {
 		return AuthInfo{}, err
 	}
 
 	treq := client.NewAccessRequest(osincli.AUTHORIZATION_CODE, areqdata)
+
```

Comment:
why do we need this ? clientId is part of the common parameters for oauth2 request